### ISSUE TITLE
[Accessibility] Tooltip is not accesible

### ISSUE THREAD

**Issue by andresgalante** [auth0/cosmos#1297]
The tooltip component is not accessible:

- The tooltip needs a `role="tooltip"`
- Add an `aria-describedby="[id]" to the element that triggers the tooltip to associate the tooltip text with the element that triggers the tooltip
- Escape: Dismisses the Tooltip.

Reference for docs: https://www.w3.org/TR/wai-aria-practices-1.1/#tooltip

Consider using popper!
